### PR TITLE
add blog halo to apps.json

### PR DESCRIPTION
**Pull request by ruibaby**
Blog developed in Java.

Doc:
[https://halo-doc.ryanc.cc/](https://halo-doc.ryanc.cc/)

GitHub:
[https://github.com/ruibaby/halo](https://github.com/ruibaby/halo)

Sites on Halo:
- https://ryanc.cc
- https://www.songhaozhi.com
- www.cnbuilder.cn
- blog.asasc.cn
- https://aodeng.cc
- http://blog.asasc.cn
- https://lnomve.com
- blog.eunji.cn
- https://kioi.cc
- https://slogc.cc
...

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -12648,6 +12648,24 @@
...
       },
       "website": "https://www.xt-commerce.com"
+    },
+    "Halo": {
+      "cats": [
+        11
+      ],
+      "html": [
+        "var halo =new \\$.halo\\(\\)",
+        "<a href=\"https://github.com/ruibaby/halo\"",
```
Comment (jvoisin) [inline src/apps.json]: What about `<a href="https://github.com/ruibaby/halo" target="_blank">Proudly published with Halo` instead, to reduc false-positives?

File: src/apps.json [hunk 2]
```diff
@@ -12648,6 +12648,24 @@
         "generator": "xt:Commerce"
       },
       "website": "https://www.xt-commerce.com"
+    },
+    "Halo": {
+      "cats": [
+        11
+      ],
+      "html": [
+        "var halo =new \\$.halo\\(\\)",
```
Comment (jvoisin) [inline src/apps.json]: Could this be moved in `"js"` instead?

File: src/apps.json [hunk 3]
```diff
@@ -12648,6 +12648,24 @@
...
+        "<a href=\"https://github.com/ruibaby/halo\"",
+        "Halo</a>"
```
Comment (jvoisin) [inline src/apps.json]: This one is too false-positive prone.

File: src/apps.json [hunk 4]
```diff
@@ -12648,6 +12648,24 @@
...
+      ],
+      "icon": "halo.svg",
+      "implies": [
+        "Java"
+      ],
+      "meta": {
+        "generator": "Halo"
```
Comment (jvoisin) [inline src/apps.json]: What about `"generator": "^Halo$"` instead?

**Comment by ruibaby** [comment]
@jvoisin I am very sorry to interrupt, I will update Halo, then modify apps.json and pull request again. I will close this pull request,Thanks!

**Pull request closed without merging**